src/obb/impl/main.cljs

```diff
@@ -47,16 +52,37 @@
   [s]
   (sci/eval-string* ctx s))
 
+(def print*
+  ;; All output from osascript by default goes to stderr. To get around this
+  ;; we use the Objective-C bridge to write directly to stdout.
+
+  ;; `delay` ensures that this import happens only when needed, and only once.
+  (let [import (delay (.import js/ObjC "Foundation"))]
+    (fn [s]
+      @import
+      (-> (.dataUsingEncoding (js/ObjC.wrap s) js/$.NSUTF8StringEncoding)
+          (js/$.NSFileHandle.fileHandleWithStandardOutput.writeData)))))
+
+(defn prn
+  [x]
+  (let [object-specifier-tag "#org.babashka.obb/object-specifier"]
```

Comment:
@borkdude Think I should define `object-specifier-tag` at the top level and refer to it in the test?

Reply:
The tests do not directly access the CLJS code but are written as integration tests, which call the obb script by "shelling out". So this is fine.